### PR TITLE
Remove omniture clickstream tracking

### PR DESCRIPTION
**Pull request by dominickendrick**
## What does this change?

This removes general Omniture click tracking from dotcom
## What is the value of this and can you measure success?

Help us migrate away from Omniture
## Does this affect other platforms - Amp, Apps, etc?

Omniture

## QA

Check that internal and external and in page link interactions do not fire. Note that Pageview events still fire
## Request for comment

@guardian/dotcom-platform @mkopka

### CODE REVIEW

**Comment by markjamesbutler** [comment]
LGTM 👍

File: static/test/javascripts/spec/common/analytics/interaction-tracking.spec.js [hunk 1]
```diff
@@ -14,21 +14,14 @@ define([
                     trackSamePageLinkClick: sinon.spy(),
                     trackExternalLinkClick: sinon.spy()
                 })
-                .mock('common/modules/analytics/omniture', {
-                    go: sinon.spy(),
-                    trackSamePageLinkClick: sinon.spy(),
-                    trackExternalLinkClick: sinon.spy()
```
Comment (NataliaLKB) [inline static/test/javascripts/spec/common/analytics/interaction-tracking.spec.js]: Does GA/Ophan still track external link clicks without omniture?

Comment (dominickendrick) [inline static/test/javascripts/spec/common/analytics/interaction-tracking.spec.js]: This is being deprecated in GA, and it doesn't happen in omniture

Comment (NataliaLKB) [inline static/test/javascripts/spec/common/analytics/interaction-tracking.spec.js]: Sorry Dom, I don't follow. With this PR, will the page still wait to register clicks to external websites so we don't lose that data?

I remember chatting about this with @gtrufitt a couple weeks ago. Then it was something that would have to be implemented before Omniture is taken out. It is pretty important data to capture and I just want to make sure we aren't going to lose it.

Comment (dominickendrick) [inline static/test/javascripts/spec/common/analytics/interaction-tracking.spec.js]: @mkopka says that we rarely report on external page tracking. The comment left by Chris Birchell about re-impimenting the [500ms delay](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/analytics/interaction-tracking.js#L72) is untrue at the moment, as we have [explicitly turned this off in Omniture](https://github.com/guardian/frontend/blob/master/static/public/javascripts/vendor/omniture.js#L16). We will continue to report external link tracking in the same way as we do now in GA, so no data will be lost. I'm looking to implement this using the [Beacon API](https://developer.mozilla.org/en-US/docs/Web/API/Beacon_API) as a seperate PR to improve accuracy, and retain performance.

**Pull request merged**